benchexec/tablegenerator/react-table/src/tests/QuantilePlot.test.js

```diff
@@ -65,25 +78,29 @@ files
       );
 
       describe("Quantile Plot should match HTML snapshot", () => {
-        setURLParameter({ plot: plotInstance.plotOptions.quantile });
+        updateURLParams({ plot: plotInstance.plotOptions.quantile });
 
         it.each(selectionResultInput)(
           "with selection of the type %s and %s results",
           (selection, results) => {
-            setURLParameter({ selection: selection.value, results });
+            updateURLParams({ selection: selection.value, results });
+
             plotInstance.refreshUrlState();
             expect(plot).toMatchSnapshot();
           },
         );
       });
 
       describe("Direct Plot should match HTML snapshot", () => {
-        setURLParameter({ plot: plotInstance.plotOptions.direct });
+        console.log(window.location.href);
+        updateURLParams({ plot: plotInstance.plotOptions.direct });
+        console.log(window.location.href);
```

Comment:
```suggestion
        updateURLParams({ plot: plotInstance.plotOptions.direct });
```
Let's remove this if everything is working now in order to not clutter the output?

Reply:
Sure sir! I forgot to remove it post-debugging.